dbusmock/mockobject.py

```diff
@@ -607,6 +607,21 @@ def AddTemplate(self, template: str, parameters: PropsType) -> None:
         self._template = template
         self._template_parameters = parameters
 
+    def _emit_signal(self, interface: str, name: str, signature: str, sigargs: Tuple[Any, ...]) -> None:
+        # pylint: disable=protected-access
+        if not interface:
+            interface = self.interface
+
+        args = _convert_args(signature, sigargs)
+
+        sig = dbus.lowlevel.SignalMessage(self.path, interface, name)
+        sig.append(*args, signature=signature)
+
+        for location in self.locations:
```

Comment:
Note to self: This is https://dbus.freedesktop.org/doc/dbus-python/dbus.service.html#dbus.service.FallbackObject.locations